modules/heat_conduction/include/materials/GapConductance.h

```diff
@@ -88,6 +88,7 @@ class GapConductance : public Material
   const VariableValue & _gap_temp_value;
   MaterialProperty<Real> & _gap_conductance;
   MaterialProperty<Real> & _gap_conductance_dT;
+  MaterialProperty<Real> & _gap_conductivity_property;
```

Comment:
Oh! Well you might consider changing the other member. It's just there as a bulk property right? Do what you think is best.